ibc/modules/src/core/ics23_commitment/commitment.rs

```diff
@@ -61,7 +62,7 @@ impl From<Vec<u8>> for CommitmentRoot {
 #[derive(Clone, Debug, PartialEq)]
 pub struct CommitmentPath;
 
-#[derive(Clone, PartialEq, Eq, Serialize)]
+#[derive(Clone, PartialEq, Eq, Serialize, Deserialize, JsonSchema)]
```

Comment:
The contract fails to build because it complains about the `JsonSchema`:

```
65 | #[derive(Clone, PartialEq, Eq, Serialize, Deserialize, JsonSchema)]
   |                                                        ^^^^^^^^^^ use of undeclared crate or module `schemars`
```

Any ideas what the problem might be?

Reply:
I think schemars itself is also needed and enabling it with this with the cosmwasm feature should help keep things compatible with Composable's other crates.
```
-use cosmwasm_schema::schemars::JsonSchema;
+#[cfg(feature = "cosmwasm")]
+use cosmwasm_schema::schemars::{self, JsonSchema};
```

```
-#[derive(Clone, PartialEq, Eq, Serialize, Deserialize, JsonSchema)]
+#[derive(Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[cfg_attr(feature = "cosmwasm", derive(JsonSchema))]
```

Reply:
Thank you, @misko9; that did the trick! I think the PR is now ready for review!